scripts/gulp-start-gae-devserver.js

```diff
@@ -0,0 +1,87 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
```

Comment:
Oh, I see -- OK, let's just add a comment to attribute the source for now, then. For the benefit of future developers, the comment should also explain why we don't use it directly.